test/modules/packages/distributedIters/DistributedIters.chpl

```diff
@@ -17,59 +17,332 @@
  * limitations under the License.
  */
 
- /*
-   This module contains iterators that can be used to distribute a `forall`
-   loop for a range or domain.
-
-   Part of a 2017 Cray summer intern project by Sean I. Geronimo Anderson
-   (github.com/s-geronimoanderson) as mentored by Ben Harshbarger
-   (github.com/benharsh).
- */
+/*
+  This module contains iterators that can be used to distribute a `forall`
+  loop for a range or domain by dynamically splitting iterations between
+  locales.
+
+  ..
+    Part of a 2017 Cray summer intern project by Sean I. Geronimo Anderson
+    (github.com/s-geronimoanderson) as mentored by Ben Harshbarger
+    (github.com/benharsh).
+*/
 module DistributedIters
 {
 use DynamicIters,
     Time;
 
-// Toggle debugging output and per-locale performance timing.
+/*
+  Toggle debugging output.
+*/
 config param debugDistributedIters:bool = false;
+
+/*
+  Toggle per-locale performance timing and output.
+*/
 config param timeDistributedIters:bool = false;
 
-// Toggle writing system information.
+/*
+  Toggle invocation information output.
+*/
 config const infoDistributedIters:bool = false;
 
+// Distributed Dynamic Iterator.
+/*
+  :arg c: The range (or domain) to iterate over. The range (domain) size must
+    be positive.
+  :type c: `range(?)` or `domain`
+
+  :arg chunkSize: The chunk size to yield to each thread. Must be positive.
```

Comment:
We tend to prefer the term ``task`` over ``thread``.

Reply:
Missed that one! :)